From 1b3298489cf561dac4efef244cf4247bc7e2d698 Mon Sep 17 00:00:00 2001
From: pengy <quic_pengy@quicinc.com>
Date: Tue, 27 Aug 2024 17:33:57 +0800
Subject: [PATCH] lttng-modules: Fix compile issue for msm-kernel 6.1

use vm_flags_set for msm-kernel version after 6.1 to solve
vm_flags type mismatch issue.

Upstream-Status: Inappropriate [other]

Change-Id: I97b2567cfee5bfcb3c39f018a242b29f3eda0618
---
 include/wrapper/mm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/wrapper/mm.h b/include/wrapper/mm.h
index 61ac812..bdf8f6f 100644
--- a/include/wrapper/mm.h
+++ b/include/wrapper/mm.h
@@ -13,7 +13,7 @@
 
 #include <lttng/kernel-version.h>
 
-#if (LTTNG_LINUX_VERSION_CODE >= LTTNG_KERNEL_VERSION(6,3,0))
+#if (LTTNG_LINUX_VERSION_CODE >= LTTNG_KERNEL_VERSION(6,1,0))
 static inline
 void wrapper_vm_flags_set(struct vm_area_struct *vma,
 		vm_flags_t flags)
-- 
2.25.1

